script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -387,92 +414,105 @@ Yeah... Setting that aside―――
 [sha:b24af8a27f943b551ad92a9656093b248bd4246a]{
 -- Page 269, Offset 27797.
 -- 「ですがアルクェイドとの協力だけは出来ません。
-"But cooperating with Arcueid is the one thing I cannot agree with.
+"However, cooperating with Arcueid is the one thing I cannot allow.
 }
 [sha:5658a540b35338c9add76455fc508c1ca27f9f4c]{
 -- Page 269, Offset 27798.
 -- この際ですからはっきりと忠告……いえ、警告しておきましょう。現在、この街で最も危ういのは彼女です。それは彼女が吸血鬼だから、という事ではありません。
-This is a good opportunity for me to advise you... no, to warn you. As we speak, there is none more dangerous in this city than her. This is not due to the fact that she is a vampire.
+If you're going to be involved,
+ then let me give you some advice... no, let me give you a warning.
+ As things stand, that woman is the most dangerous creature in this city.
+ That's not just because she's a vampire.
```

Comment:
The way the last two sentences place emphasis on things should be better reflected.
>As things stand, the most dangerous creature in the city is %{i}her%{/i}.
>And that's not just because she's a vampire either.